Index: test.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print(\"Start\")\r\nprint(\"Hello world\")\r\nprint(\"Hello world\")
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test.py b/test.py
--- a/test.py	(revision f038991938f18063f02b0f4a9e623210df2a362d)
+++ b/test.py	(date 1732028735791)
@@ -1,3 +1,3 @@
-print("Start")
+print("Hello world")
 print("Hello world")
-print("Hello world")
\ No newline at end of file
+#123
\ No newline at end of file
Index: .idea/Repetitors.iml
===================================================================
diff --git a/.idea/Repetitors.iml b/.idea/Repetitors.iml
deleted file mode 100644
--- a/.idea/Repetitors.iml	(revision f038991938f18063f02b0f4a9e623210df2a362d)
+++ /dev/null	(revision f038991938f18063f02b0f4a9e623210df2a362d)
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="PYTHON_MODULE" version="4">
-  <component name="NewModuleRootManager">
-    <content url="file://$MODULE_DIR$">
-      <excludeFolder url="file://$MODULE_DIR$/.venv" />
-    </content>
-    <orderEntry type="jdk" jdkName="Python 3.12 (Repetitors)" jdkType="Python SDK" />
-    <orderEntry type="sourceFolder" forTests="false" />
-  </component>
-</module>
\ No newline at end of file
